packages/fast-form-generator-react/src/form/form-item.select.tsx

```diff
@@ -53,7 +58,11 @@ class FormItemSelect extends React.Component<IFormItemSelectProps & IManagedClas
      * Handles the onChange of the select element
      */
     private handleChange = (event: React.FormEvent<HTMLSelectElement>): void => {
-        this.props.onChange(this.props.dataLocation, (event.target as HTMLSelectElement).value);
+        if (this.props.propertyType === "number") {
+            this.props.onChange(this.props.dataLocation, parseInt((event.target as HTMLSelectElement).value, 0));
```

Comment:
the base for parseInt should be 10

Reply:
I added due to a tslint error. Should we keep it or remove it and add a tslint disable comment?

Reply:
If it throws a tslint error we can leave it in, I think it's more preferable to disabling